mtags/src/main/scala-2.13.8/meta/internal/pc/Compat.scala

```diff
@@ -19,4 +25,33 @@ trait Compat { this: MetalsGlobal =>
 
   def constantType(c: ConstantType): ConstantType =
     if (c.value.isSuitableLiteralType) LiteralType(c.value) else c
+
+  def runOutline(files: OutlineFiles): Unit = {
+    this.settings.Youtline.value = true
```

Comment:
Would probably be good to extract that and have under scala-2.13 directory

Reply:
I was just remembered why I didn't do it this way. We filter the whole directory out for some cross versions:
```Scala
val scalaVersionsWithSpecialCompat =
  Set("2.13.5", "2.13.6", "2.13.7", "2.13.8")
if (scalaVersionsWithSpecialCompat(scalaVersion.value))
  current.filter(f => f.getName() != "scala-2.13")
```
Alternatives?

Reply:
Ach, we need to leave it as is then, we'll drop that part once we stop supporting those versions